src/assets/levels.json

```diff
@@ -1,11 +1,35 @@
 [
   {
     "type": "money",
+    "goal": 10010
```

Comment:
My reasoning was that as a kid I would prefer to see that I'm progressing and levels kind of indicate that. I don't know about you, but I can't reach 11000 easily?